app/assets/javascripts/govuk_publishing_components/components/layout-super-navigation-header.js

```diff
@@ -134,8 +134,8 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
     var $navMenuLinks = this.$navMenu.querySelectorAll('li a')
     var $firstNavLink = $navMenuLinks[0]
     var $lastNavLink = $navMenuLinks[$navMenuLinks.length - 1]
-    var $searchMenuLinks = this.$searchMenu.querySelectorAll('li a')
-    var $lastSearchLink = $searchMenuLinks[$searchMenuLinks.length - 1]
+    var $searchMenuTabable = this.$searchMenu.querySelectorAll('li a, input, button')
+    var $lastSearchTabable = $searchMenuTabable[$searchMenuTabable.length - 1]
```

Comment:
It makes the variable name a bit longer, but wonder if we should keep this consistent with the variable above, for example `lastSearchMenuTabbable`